packages/next-app/src/contexts/PoolMetadataContext.tsx

```diff
@@ -109,6 +115,10 @@ export function PoolMetadataProvider({ children }: { children: ReactNode }) {
     setMetadataUpdated(!isEqual(metadata, originalMetadata));
   }, [metadata, originalMetadata]);
 
+  useEffect(() => {
+    setIsMetadataValid(!metadata.some(hasNull));
```

Comment:
🤔 So even if a description is null the metadata is invalid? Not sure I think that's useful. Also, any reason not to use the zod type validation here?

Reply:
Since we established earlier that the description and value couldn't be null, I thought it would make sense to check the `description` and `value` values. 

And about the zod, I forgot to use it 😢